client/my-sites/post-type-list/post-type-list-post-actions/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EllipsisMenu from 'components/ellipsis-menu';
+import PostTypeListPostActionsEdit from './edit';
+import PostTypeListPostActionsPublish from './publish';
+import PostTypeListPostActionsTrash from './trash';
+import PostTypeListPostActionsView from './view';
+import PostTypeListPostActionsRestore from './restore';
+
+export default function PostTypeListPostActions( { globalId } ) {
+	return (
+		<div className="post-type-list-post-actions">
+			<EllipsisMenu position="bottom left">
+				<PostTypeListPostActionsView globalId={ globalId } />
```

Comment:
My experience with dropdown-menu is that arrays become inflexible and just declaratively setting the children is preferred. I'd be ok with making any children appear after an hr division and treating the default set as static. Alternatively we could have boolean props for the base four actions that you could disable if needed.